newsfragments/3448.minor

```diff
@@ -0,0 +1 @@
+Convert modules that only reference `unicode` to use `str`.
```

Comment:
Given Python 3 porting has no meaningful impact on users, you can just leave these news fragments empty (the alternative is 300 changelog items that are just noise).

Reply:
From previous feedback on a previous PR, I thought that's exactly what `*.minor` is for, stuff that won't be included in `NEWS`.  In which case I figure better to say something than nothing.  Am I misunderstanding something?

Reply:
For `.minor` fragments the ticket *number* will be included in the generated news file and the content will be discarded.  Having content doesn't hurt but we typically leave the these fragments empty since the content doesn't go anywhere.  For developers the information is all conveyed elsewhere - in the ticket, in the PR description, in the changeset itself, etc.

Reply:
Ah, my mistake.